requirements.txt

```diff
@@ -27,3 +27,6 @@ click==8.0.3
 terminaltables==3.1.0
 pluginbase==1.0.1
 sentry-sdk==0.16.3
+
+# Test
+pytest==7.0.1
```

Comment:
Does it matter this is v7.0.1 but in WFE v6.2.4 is required? Actually, do we need these packages in production builds at all?

Might be part of another task of putting the images on a diet, your call.